tests/integration-tests/tests/schedulers/test_slurm/test_slurm_pmix/pcluster.config.yaml

```diff
@@ -1,5 +1,16 @@
 Image:
   Os: {{ os }}
+{% if scheduler == "slurm" and use_login_node == True %}
+LoginNodes:
+  Pools:
+    - Name: testpool
+      InstanceType: {{ instance }}
+      Count: 2
+      Networking:
+        SubnetIds:
+          - {{ public_subnet_id }}
+      GracetimePeriod: 60
```

Comment:
Do we need `GracetimePeriod: 60` for the test or as in the others config we can set it at `GracetimePeriod: 5` ?